lambdas/tests/unit/handlers/session-handler.test.ts

```diff
@@ -493,9 +493,56 @@ describe("SessionLambda", () => {
                     clientSessionId: "test-journey-id",
                 },
                 extensions: {
-                    evidence: {
-                        context: "identity_check",
-                    },
+                    evidence: [
+                        {
+                            context: "identity_check",
+                        },
+                    ],
```

Comment:
This is currently used by Check HMRC, and not by Experian KBV

Reply:
Yeah I was looking to update documentation this morning and was going to check the other CRI's that use the TS lambdas. 

I followed the structure in the AC of the Jira ticket. Where I have seen extensions.evidence it had been an array e.g. https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3485663247/TxMA+Events#CIC-CRI
I'm happy however we want to structure it though. 

So remove the arrays and just have objects for both `evidence` and `evidence_requested`?

Reply:
I have now made `evidence_request` just an object and undone the array change for `evidence`. Confirmed this with Caitlin.
Are there any other changes needed?